tests/model_serving/model_runtime/triton/probes/conftest.py

```
@@ -0,0 +1,226 @@
+        kind_dict=template_dict,
+    ) as template:
+        yield template
+
+
+def create_triton_serving_runtime(protocol: str, triton_runtime_image: str) -> dict[str, Any]:
+    """Create Triton ServingRuntime object definition."""
+    volumes = []
+    volume_mounts = []
+    if protocol == Protocols.GRPC:
+        volumes.append({"name": "shm", "emptyDir": {"medium": "Memory", "sizeLimit": "2Gi"}})
+        volume_mounts.append({"name": "shm", "mountPath": "/dev/shm"})
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)